gulpfile.js

```diff
@@ -33,6 +33,7 @@ var nwBuilderOptions = {
     macIcns: './images/bf_icon.icns',
     macPlist: { 'CFBundleDisplayName': 'Betaflight Blackbox Explorer'},
     winIco: './images/bf_icon.ico',
+    zip: false,
```

Comment:
If we want to go with this, we need to do mcgiver requested changes on the installer side.

Reply:
Removed.